pkg/arvo/neo/cod/std/src/fil/counter.txt

```diff
@@ -294,17 +307,19 @@ A `note` is one of the four types of command any shrub will accept.
 Let’s `%make` a shrub at path `/foo/bar` from the Dojo, giving it an initial state of `0`. We’ll explain the structure of the `%make` note in more detail in the Diary tutorial.
 
 ```
-:neo &neo-card [~[[%p our] %foo %bar] [%make %counter `[%number !>(0)] ~]]
+:neo &neo-card [~[[%p our] %counter] [%make %counter `[%number !>(0)] ~]]
 ```
 
-You should see `>> %make /foo/bar` in the Dojo if successful.
+You should see `>>  /~zod/~/sys -> /~zod/counter: make` in the Dojo if successful.
 
 Now we can now send a `%poke` to the counter shrub at this path.
 
 ```
-:neo &neo-card [~[[%p our] %foo %bar] [%poke [%counter-diff !>([%inc ~])]]]
+:neo &neo-card [~[[%p our] %counter] [%poke [%counter-diff !>([%inc ~])]]]
 ```
 
+TODO: How can the reader verify that the poke worked?
```

Comment:
A while ago Liam removed his shrub shell which gave you the ability to inspect a shrub's state from the terminal, no replacement for it so far. Currently the only way to inspect the value at a path is to surface it in Sky. Should be a DevEx priority to get something like this back in.